modules/dist/src/main/scripts/build-deploy.xml

```diff
@@ -353,6 +353,7 @@
       <copy todir="@{targetdir}/org/apache/cxf/main" flatten="false" overwrite="true">
         <fileset dir="@{thirdpartydir}/lib">
           <include name="**/cxf-core*.jar"/>
+          <include name="**/cxf-rt-features-logging*.jar"/>
```

Comment:
Is it really needed in main/core module?

Reply:
I discussed this with Alessio and we decided to put this dependency in org/apache/cxf/main/ folder.

Reply:
Ok then ;)